### PR TITLE
[KHWEBA-8] add tag dialog

### PR DESCRIPTION
**Pull request by gift-a**
image

### CODE REVIEW

**Comment by ihor7016** [comment]
Please use input without label but with placeholder and hint underneath (like in transactions dialog)

**Comment by ihor7016** [comment]
BTW buttons should be Cancel and Ok (please also rename methods in js according to this changes)

File: src/components/app/app.scss [hunk 1]
```diff
@@ -1,3 +1 @@
-.app {
-  color: red;
-}
+@import url("https://fonts.googleapis.com/css?family=Roboto:300,400,500");
```
Comment (ihor7016) [inline src/components/app/app.scss]: move this import to html

**Review by ihor7016 — DISMISSED** [review]
~~will be merged after the drawer~~
UPD -- please fix webpack and importants !!! see https://github.com/ihor7016/buxy-vanilla/pull/2#pullrequestreview-102879082

**Comment by gift-a** [comment]
ready

**Review by ihor7016 — APPROVED** [review]

**Pull request merged**